flexget/plugins/search_sceneaccess.py

```diff
@@ -234,6 +235,14 @@ def search(self, task, entry, config=None):
             Search for entries on SceneAccess
         """
 
+        try:
+            session = task.requests
+        except:
```

Comment:
catching everything like this is ill-advised. I also don't see it ever throwing an exception. You're accessing an object member that is set on object creation.